### PR TITLE
DIV-5762 - Added div-hm configs

### PR DESCRIPTION
**Pull request by hosainnet**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines

### JIRA link (if applicable) ###

DIV-5762

### Change description ###

To replace the divorce health monitor app with an AKS deployment

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/aat/common/divorce/div-hm.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+spec:
+  releaseName: div-hm
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: div-hm
+    version: 1.1.0
+  values:
+    java:
```
Comment (timja) [inline k8s/aat/common/divorce/div-hm.yaml]: isn't  this a nodejs app?

File: k8s/aat/common/divorce/div-hm.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: div-hm
+  namespace: divorce
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
```
Comment (timja) [inline k8s/aat/common/divorce/div-hm.yaml]: isn't this a nodejs app?

Comment (timja) [reply k8s/aat/common/divorce/div-hm.yaml]:
```suggestion
    flux.weave.works/tag.nodejs: glob:prod-*
```

File: k8s/prod/common/divorce/div-hm.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: div-hm
+  namespace: divorce
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
```
Comment (timja) [inline k8s/prod/common/divorce/div-hm.yaml]:
```suggestion
    flux.weave.works/tag.nodejs: glob:prod-*
```

File: k8s/aat/common/divorce/div-hm.yaml (hunk 2 above)
Comment (hosainnet) [reply k8s/aat/common/divorce/div-hm.yaml]: 👍

File: k8s/aat/common/divorce/div-hm.yaml [hunk 4]
```diff
@@ -27,7 +27,7 @@ spec:
       readinessPeriod: 15
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
-      image: hmctspublic.azurecr.io/div/hm:prod-468cf5ac
+      image: hmctspublic.azurecr.io/div/hm:prod-c1804250
```
Comment (timja) [inline k8s/aat/common/divorce/div-hm.yaml]: i would just put latest and let flux find it for you

File: k8s/prod/common/divorce/div-hm.yaml [hunk 5]
```diff
@@ -27,7 +27,7 @@ spec:
       readinessPeriod: 15
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "baaed5d0-7666-4dce-bf74-edfceeb566f9"
-      image: hmctspublic.azurecr.io/div/hm:prod-468cf5ac
+      image: hmctspublic.azurecr.io/div/hm:prod-c1804250
```
Comment (timja) [inline k8s/prod/common/divorce/div-hm.yaml]: let's make it work on AAT first before we add prod

**Comment by hosainnet** [comment]
> Can you add CODEOWNERS change please

Ah thought we had /div-*/

File: k8s/prod/common/divorce/div-hm.yaml (hunk 5 above)
Comment (hosainnet) [reply k8s/prod/common/divorce/div-hm.yaml]: 👍

File: CODEOWNERS [hunk 6]
```diff
@@ -61,6 +61,7 @@ k8s/**/common/divorce/div-cms.yaml @hmcts/divorce
 k8s/**/common/divorce/div-dgs.yaml @hmcts/divorce
 k8s/**/common/divorce/div-cos.yaml @hmcts/divorce
 k8s/**/common/divorce/div-fps.yaml @hmcts/divorce
+k8s/**/common/divorce/div-hm.yaml @hmcts/divorce
```
Comment (timja) [inline CODEOWNERS]:
```suggestion
k8s/aat/common/divorce/div-hm.yaml @hmcts/divorce
```

**Pull request merged**